src/__test__/index.test.ts

```diff
@@ -7,13 +7,13 @@
 import { expect } from 'chai'
 import { NewStoreReply } from '@textile/threads-client-grpc/api_pb'
 import { WriteTransaction } from 'src/WriteTransaction'
+import { ReadTransaction } from 'src/ReadTransaction'
 import { Client } from '../index'
 import { JSONQuery, JSONOperation } from '../models'
 import { Where } from '../query'
-import { ReadTransaction } from 'src/ReadTransaction'
 
 const sleep = (ms: number) => new Promise(r => setTimeout(r, ms))
-const client = new Client('http://localhost:6007')
+const client = new Client('http://127.0.0.1:6107')
```

Comment:
this is a kinda ugly change since you'll have to manually override local daemon anytime you want threads to pass. 

i'd setup threads to use an ENV var if available. otherwise, use the standard default port. then in CI you can just set that ENV var. E.g.,

https://github.com/textileio/dart-threads-client/blob/master/test/threads_client_test.dart#L45

Reply:
just realizing that's trickier here. so yeah, i'd try to keep the tests running against the standard default port

Reply:
Naw this is a good observation, I'll just use the default port. No need to introduce ENV vars here, as they won't do anything in browser anyway. Default port is fine.